galasa-parent/dev.galasa.framework.api.ras/src/main/java/dev/galasa/framework/api/ras/internal/routes/ResultNamesRoute.java

```diff
@@ -49,10 +47,11 @@ public HttpServletResponse handleRequest(String pathInfo, QueryParameters queryP
 
     public String retrieveResults (QueryParameters queryParams) throws ServletException, InternalServletException{
```

Comment:
Can we pass a RasQueryParameters in as the parameter instead ? Not sure by reading this code in a review as missing context.

Reply:
Then we wouldn't need rQueryParams as they would just be queryParams perhaps.

Reply:
So this method is an override, should I change it in the main one to do this?

Reply:
Added new function to handle getSortValue with and without a default value in RasQueryParameters